packages/nimble-components/karma.conf.headless.js

```diff
@@ -2,12 +2,14 @@
  * A karma config file that sets up reporting for headless test runs
  */
 
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
After https://github.com/ni/javascript-styleguide/pull/184 this will be caught and removed as unneeded by eslint